crates/kitsune-captcha/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "kitsune-captcha"
+version.workspace = true
+edition = "2021"
+
+[dependencies]
+async-trait = "0.1.73"
+enum_dispatch = "0.3.12"
+http = "0.2.9"
+http-body = "0.4.5"
+hyper = { version = "0.14.27", features = [
+    "client",
+    "deprecated",
+    "http1",
+    "http2",
+    "runtime",
+] }
+hyper-rustls = { version = "0.24.1", features = ["http2"] }
+kitsune-http-client = { path = "../kitsune-http-client" }
+serde = { version = "1.0.183", features = ["derive"] }
+serde_qs = "0.12"
```

Comment:
Do we need support for nested URL parameters? If not, please replace this with `serde_urlencoded`